plan_test.go

```diff
@@ -0,0 +1,71 @@
+package plan
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/terraform"
+	"os"
+	"testing"
+)
+
+func OpenReadPlan(testCase string) *terraform.Plan {
+	file, err := os.Open(testCase)
+	if err != nil {
+		fmt.Errorf("unexpected error %v", err)
+	}
+
+	plan, err := terraform.ReadPlan(file)
+	if err != nil {
+		fmt.Errorf("unexpected error %v", err)
+	}
+
+	return plan
+}
+
+func TestIsDestroyedCreate(t *testing.T) {
+	isDestroyed, _ := IsDestroyingEBSVolume(OpenReadPlan("create.plan"))
```

Comment:
All test cases need also testing of name that are returned in the second result (where you are using `_` right now)